### PR TITLE
Issue658 update min max limits and issue 625 add missing activate inputs 

### PR DESCRIPTION
**Pull request by EttoreZ**
This PR is for #658 and #625.

### CODE REVIEW

File: testcases/bestest_air/models/BESTESTAir/TestCases/TestCase_Ideal.mo [hunk 1]
```diff
@@ -769,6 +769,12 @@ see https://www.eia.gov/environment/emissions/co2_vol_mass.php.
 revisions="<html>
 <ul>
 <li>
+October 18, 2024, by Ettore Zanetti:<br/>
+Update <code>min</code> and <code>max</code> for ovewrite blocks.
```
Comment (dhblum) [inline testcases/bestest_air/models/BESTESTAir/TestCases/TestCase_Ideal.mo:773]: Specify for which overwrite blocks.

File: testcases/bestest_hydronic/models/BESTESTHydronic/TestCase.mo [hunk 2]
```diff
@@ -577,6 +589,18 @@ https://www.eia.gov/environment/emissions/co2_vol_mass.php</a>
 <ul>
 <li>
 October 18, 2024, by Ettore Zanetti:<br/>
+Update <code>min</code> and <code>max</code> for ovewrite blocks.
```
Comment (dhblum) [inline testcases/bestest_hydronic/models/BESTESTHydronic/TestCase.mo:592]: Specify for which overwrite blocks.

File: testcases/bestest_hydronic_heat_pump/models/BESTESTHydronicHeatPump/TestCase.mo [hunk 3]
```diff
@@ -879,6 +889,18 @@ https://www.carbonfootprint.com/docs/2019_06_emissions_factors_sources_for_2019_
 <ul>
 <li>
 October 18, 2024, by Ettore Zanetti:<br/>
+Update <code>min</code> and <code>max</code> for ovewrite blocks.
```
Comment (dhblum) [inline testcases/bestest_hydronic_heat_pump/models/BESTESTHydronicHeatPump/TestCase.mo:892]: I don't see that this was done for bestest_hydronic_heat_pump.

**Review by dhblum — CHANGES_REQUESTED** [review]
Thanks @EttoreZ.  Please address inline comments.

File: testcases/bestest_air/models/BESTESTAir/TestCases/TestCase_Ideal.mo (hunk 1 above)
Comment (EttoreZ) [reply testcases/bestest_air/models/BESTESTAir/TestCases/TestCase_Ideal.mo]: Updated with the overwrite block names

File: testcases/bestest_hydronic/models/BESTESTHydronic/TestCase.mo (hunk 2 above)
Comment (EttoreZ) [reply testcases/bestest_hydronic/models/BESTESTHydronic/TestCase.mo]: Updated with the overwrite block names

File: testcases/bestest_hydronic_heat_pump/models/BESTESTHydronicHeatPump/TestCase.mo (hunk 3 above)
Comment (EttoreZ) [reply testcases/bestest_hydronic_heat_pump/models/BESTESTHydronicHeatPump/TestCase.mo]: You are right. I updated the release notes removing this.

**Review by dhblum — APPROVED** [review]

**Pull request merged**